public/angular/templates/users/modal.html

```diff
@@ -62,10 +62,10 @@ <h4 class="modal-title">
             </div>
           </div>
           <div ng-show="user._id == currentUser._id">
-            <div class="form-group" ng-hide="showPassword || form.password.$dirty || form.passwordConfirmation.$dirty">
-              <button type="button" class="btn col-lg-6 col-lg-offset-3" ng-click="showPassword = !showPassword" translate>Change password</button>
+            <div class="form-group" ng-hide="model.showPassword || form.password.$dirty || form.passwordConfirmation.$dirty">
+              <button type="button" class="btn col-lg-6 col-lg-offset-3" ng-click="model.showPassword = true" translate>Change password</button>
```

Comment:
That wouldn't do anything; ng-click takes an expression to evaluate. I suppose we could write a showPassword function.